syntax = "proto2";
package ndn_message;

// Protobuf has no "outer" message type, so we need to put the TLV RepoCommandParameter
// message inside an outer "typeless" RepoCommandParameterMessage.

message RepoCommandParameterMessage {
  message Name {
    repeated bytes component = 8;
  }

  message RepoCommandParameter {
    optional Name name = 7;
    optional uint64 process_id = 206;
  }

  // A RepoCommandParameterMessage has one RepoCommandParameter.
  required RepoCommandParameter repo_command_parameter = 201;
}